src/state/store.js

```diff
@@ -77,16 +78,8 @@ export function getFromQueryStringOrDefault(fieldKey, isMulti = false) {
 }
 
 const initialState = {
-  probe: {
```

Comment:
can you give some more detail about why you pulled probe out?

Reply:
Sure! The short answer is that the current probe can be inferred from the path and synchronizing the URL (which is something like a global store itself) and the `store` object seemed fraught and unnecessary.

That said, this approach isn't perfect either. I'm very open to discussing this further.

For the time being, see `currentProbe` in *telemetry-search.js* which partly replaces this.

Reply:
I'm not averse to this, just curious. It seems like if we move to page-based views, we can have less stuff in a global store. Maybe each view has its own store / query string to capture what it needs. Seems like this is a step in that direction anyway.